docs/data-flows-catalogue.md

```diff
@@ -0,0 +1,24 @@
+#Data flows for data catalogue staging areas
+
+We use staging areas for data managers to upload their network or cohort variable metadata and cohort mappings.
+Data managers of cohorts get access to databases on a server to upload their data using templates in a preloaded model, which is a derivative of the [full data catalogue model](https://github.com/molgenis/molgenis-emx2/blob/master/data/datacatalogue/molgenis.csv): 
+Links to the models and how to set up staging areas for the data cataloge are [here](https://github.com/molgenis/molgenis-py-catalogue-transform/datamodels)
```

Comment:
suggest to move the link from 'here' to 'models and how to set up staging areas for the data cataloge' to allow use the scan for links